index.php

```diff
@@ -17,9 +17,7 @@
                 <p>Total queries (<span id="unique_clients">-</span> clients)</p>
                 <h3 class="statistic"><span id="dns_queries_today">---</span></h3>
             </div>
-            <div class="icon">
-                <i class="ion ion-earth"></i>
-            </div>
+            <span class="icon fa fa-globe"></span>
```

Comment:
These changes result in a broken dashboard:
![screenshot from 2017-11-16 22-57-07](https://user-images.githubusercontent.com/4417660/32928809-e88d5be4-cb21-11e7-8dfa-fa562cf87803.png)

Reply:
You're still using the old styles. Hard refresh to fix.

Reply:
The "Domains on Blocklist" section seems to have the icon on top of the text:
![screenshot from 2017-11-17 12-37-38](https://user-images.githubusercontent.com/4417660/32960773-2c69b764-cb94-11e7-8bf7-77a15cde24c5.png)